### ISSUE TITLE
node_modules/micromark-util-decode-numeric-character-reference/index.js 23:11 Module parse failed: Identifier directly after number

### ISSUE THREAD

**Issue by MisterRichards** [microsoft/BotFramework-WebChat#5006]
### Is it an issue related to Adaptive Cards?

No

### Is this an accessibility issue?

No

### What version of Web Chat are you using?

Latest production

### Which distribution are you using Web Chat from?

NPM

### Which hosting environment does this issue primarily affect?

Web apps

### Which browsers and platforms do the issue happened?

Others or unrelated

### Which area does this issue affect?

Development experience

### Please describe the bug

Started getting this error.
node_modules/micromark-util-decode-numeric-character-reference/index.js 23:11 Module parse failed: Identifier directly after number

Tracked it down to here:
https://github.com/micromark/micromark/issues/161

I feel the attitude of the developers over on Micromark is absolute crap.  

I came back to our project, and **downgraded *your* library to version "4.15.9", and everything started compiling fine again.**

So apparently, Micromark is some new dependency for you guys?... you might think twice leaning on them based on their attitude to people who are just reporting the issues the library is causing.

### How to reproduce the issue?

1. Include "botframework-webchat": "4.16.0", in your project
2. Build your project with any version of webpack 4.
3. Boom!  The build blows up in your face.

### What do you expect?

I would expect the project to continue to build successfully using your latest version.

### What actually happened?

Previously, we had relied on ^ to automatically upgrade your library.  It seems your most recent version introduces a requirement (thanks to MicroMark) for WebPack 5.  That's a pretty big dependency change and should probably be marked with a major version upgrade.

**Comment by manish-chi** [comment]
Hi Even I am getting this error. Is there any fix? Thanks.

**Comment by OEvgeny** [comment]
We have Web Chat uses with Webpack 4 / CRA. We have tests on Webpack 5 latest. Have you tried the latest Web Chat release from main? If this doesn't help full repro is required to track down the issue. We can't test every combination of old/obsolete tools and their plugins. Thanks for understanding.

**Comment by MisterRichards** [comment]
> We have Web Chat uses with Webpack 4 / CRA. We have tests on Webpack 5 latest. Have you tried the latest Web Chat release from main? If this doesn't help full repro is required to track down the issue. We can't test every combination of old/obsolete tools and their plugins. Thanks for understanding.

Yes, understood.  You can kinda tell by the fact that this issue is 8 months old that I already moved on.
